notebooks/tflite-selfie-segmentation/tflite-selfie-segmentation.ipynb

```diff
@@ -87,20 +87,23 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        if not Path("CEO%20Pat%20Gelsinger%20on%20Leading%20Intel.mp4").exists():
</pre><p>the same comment about name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2535/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>